config/benchmarkssuperglue/gap_bert.conf

```diff
@@ -0,0 +1,39 @@
+// This imports the defaults, which can be overridden below.
+include "../defaults.conf"  // relative path to this file
+
+project_dir = ${JIANT_PROJECT_PREFIX}
+exp_name = "bert_gap_running"  // configure this
+run_name = "run"  // default
+
+pretrain_tasks = "gap-coreference"  // empty: don't run main training phase
+target_tasks = "gap-coreference"   // train classifier only
+
+// Eval will use task-specific params.
+do_pretrain = 1       // skip main train phase
+do_target_task_training = 1  // train using eval task params
+do_full_eval = 1
+write_preds = "val,test"
+allow_reuse_of_pretraining_parameters = 1 
+
+reload_tasks = 1
+reload_vocab = 1
+reload_index = 1
+reindex_tasks = "gap-coreference"
+load_model = 1
+bert_model_name = "bert-base-cased"
+tokenizer = ${bert_model_name}
+bert_fine_tune = 1
+batch_size = 8
+max_seq_len = 511
+cove = 0
+word_embs = "none"
+elmo = 0
+
+sent_enc = "null"
+skip_embs = 1  // forward embeddings from lower level. 
+sep_embs_for_skip = 1  // use task embeddings since we skip the generic ones. 
```

Comment:
In the task-specific classifier, sep_embs_for_skip is used to use the task-specific embedding

Reply:
Not sure I understand. Isn't everything task-specific with BERT?

Reply:
This shouldn't make a difference with BERT or OpenAI.GPT, just elmo.